### DISCUSSION TITLE
Error on startup of audio kit

### DISCUSSION THREAD

**Discussion by livingrainman** [marcel-licence/esp32_midi_sampler#67] [Q&A]
After encountering the es8388 codec failure message on startup, I changed the I2C configuration in the boards section of the library to "2" from "1" but now I get the following error: 

E (1814) vfs_fat_sdmmc: mount_to_vfs failed (0xffffffff).
E (2878) vfs_fat_sdmmc: mount_to_vfs failed (0xffffffff).
E (3943) vfs_fat_sdmmc: mount_to_vfs failed (0xffffffff).
Firmware started successfully
[?25l[0;0HCard Mount Failed  

I am relatively new to the esp32 and would love to get your lovely code working.  Any help would be greatly appreciated.  Thanks!

**Comment by marcel-licence** [comment]
Hey,

looks like you run out of memory. You could try to make for example the delay effect shorter/smaller. It might be also possible that the micro dip switch is not setup correctly. Did you check the switches?

Best regards,

**Reply by livingrainman** [reply]
Thanks for the reply!  I now have everything loading properly and have enabled the midi stream player as a test.  The serial port output shows me that the midi stream does play but I am not getting any audio from the headphone port on the audio kit.  Almost there!  Any help would be great.  Thanks.

**Reply by marcel-licence** [reply]
It might be possible that the I2S setting is incorrect. I2C is just the communication with the audio codec / controlling and setup of the codec. I2S is only the data which cannot be checked by the firmware.
